### PR TITLE
fix:monthview content missing while draging all-day schedule(fix #673)

### LINKED ISSUE #673: month-view: content missing while draging all-day schedule
## Describe the bug
In month view, when you drag an all-day schedule, the floating div will lose some content (will be empty in some case)

## To Reproduce
1. Go to month view
2. Create an all-day schedule
3. Drag it.

## Expected behavior
The floating div following the mouse should have the same content as the schedule

## Screenshots
captured (2)

## Desktop (please complete the following information):
 - OS: 10.15.6
 - Browser chrome
 - 84.0.4147.89

## Additional context
the html of the floating div:
```html
<div class="tui-full-calendar-month-move-guide tui-full-calendar-weekday-schedule-title >
             1
    </div>
</div>
<div class=" tui-full-calendar-month-guide-cover"="" style="height:24px; border-radius: 2px;"></div>
```
By checking the code, I found that the bug seems come from ```moveGuide.hbs```
```hbs
<div class="{{CSS_PREFIX}}month-move-guide {{CSS_PREFIX}}weekday-schedule-title {{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused"{{/unless}}>
        {{#if model.isAllDay}}
            {{{allday-tmpl model}}}
        {{else}}
            {{{time-tmpl model}}}
        {{/if}}
    </div>
```
If it is an all-day schedule, the closing```"``` of ```class```will be missed.
I will try to make a PR to fix it :)

### PR DESCRIPTION
**Pull request by lhysdl**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
fix #673

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

File: src/js/handler/month/moveGuide.hbs [hunk 1]
```diff
@@ -13,7 +13,7 @@
     {{#unless model.isAllDay}}
     <span class="{{CSS_PREFIX}}weekday-schedule-bullet {{CSS_PREFIX}}weekday-schedule-bullet-focused" style="top: {{styles.scheduleBulletTop}}px;"></span>
     {{/unless}}
-    <div class="{{CSS_PREFIX}}month-move-guide {{CSS_PREFIX}}weekday-schedule-title {{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused"{{/unless}}>
+    <div class="{{CSS_PREFIX}}month-move-guide {{CSS_PREFIX}}weekday-schedule-title {{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused{{/unless}}">
```
Comment (jungeun-cho) [inline src/js/handler/month/moveGuide.hbs:16]: `{{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused{{/unless}}"`
I think you can erase it. It is not used.
The style is being overridden.
https://github.com/nhn/tui.calendar/blob/master/src/css/weekday.styl#L108

**Comment by jungeun-cho** [comment]
@dongsik-yoo Please check this PR.

**Comment by jungeun-cho** [comment]
ok to test

**Comment by lhysdl** [comment]
@jungeun-cho how can I see the detail of the failing check ?

**Comment by jungeun-cho** [comment]
ok to test

**Pull request merged**